lib/wholes.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Whole do
+  def counter(li, co) do
+    Enum.map(co..1, fn n -> Enum.reduce(li, 0, fn x, acc -> acc + Kernel.abs(n - x)  end) end)
```

Comment:
Since `1.12`, reversed ranges require an explicit step, otherwise, there would be a warning thrown by a compiler.
```suggestion
    Enum.map(co..1//-1, fn n -> Enum.reduce(li, 0, fn x, acc -> acc + Kernel.abs(n - x)  end) end)
```
Why would we need it reversed though?